app/views/catagories/index.html.erb

```diff
@@ -0,0 +1,62 @@
+<div style='background-color: #dedede; height: 100vh;'>
+
+
```

Comment:
Please follow design guidelines for Typography and colors.

Reply:
Thanks for the recommendations @SajjadAhmad14 .
Actually, for the colors, I am using the colors required in the project.
I don't think there is problem with colors